plugins/aws/forecast/datasetExportEncrypted.js

```diff
@@ -0,0 +1,107 @@
+    more_info: 'In AWS Forecast, you can save forecast reports on S3 in CSV format. Make sure to encrypt these export before writing them to the bucket in order to follow your organizations\'s security and compliance requirements.',
+    recommended_action: 'Create Forecast exports with encryption enabled',
+    link: 'https://docs.aws.amazon.com/forecast/latest/dg/howitworks-forecast.html',
+    apis: ['ForecastService:listForecastExportJobs', 'KMS:listKeys', 'KMS:describeKey'],
+    settings: {
+        forecast_dataset_desired_encryption_level: {
+            name: 'Forecast Dataset Desired Encryption Level',
+            description: 'In order (lowest to highest) awskms=AWS managed KMS; awscmk=Customer managed KMS; externalcmk=Customer managed externally sourced KMS; cloudhsm=Customer managed CloudHSM sourced KMS',
+            regex: '^(awscmk|externalcmk|cloudhsm)$',
+            default: 'awscmk'
+        }
+    },
+
+    run: function(cache, settings, callback) {
+        var results = [];
+        var source = {};
+        var regions = helpers.regions(settings);
+
+        var config = {
+            desiredEncryptionLevelString: settings.forecast_dataset_desired_encryption_level || this.settings.forecast_dataset_desired_encryption_level.default
+        };
+
+        var desiredEncryptionLevel = helpers.ENCRYPTION_LEVELS.indexOf(config.desiredEncryptionLevelString);
+        var currentEncryptionLevel;
+
+        async.each(regions.forecastservice, function(region, rcb){
+            var listForecastExportJobs = helpers.addSource(cache, source,
+                ['forecastservice', 'listForecastExportJobs', region]);
+
+            if (!listForecastExportJobs) return rcb();
+
+            if (listForecastExportJobs.err || !listForecastExportJobs.data) {
+                helpers.addResult(results, 3,
+                    'Unable to query forecast exports: ' + helpers.addError(listForecastExportJobs), region);
+                return rcb();
+            }
+
+            if (!listForecastExportJobs.data.length) {
+                helpers.addResult(results, 0, 'No forecast exports found', region);
+                return rcb();
+            }
+
+            var listKeys = helpers.addSource(cache, source,
+                ['kms', 'listKeys', region]);
+
+            if (!listKeys || listKeys.err || !listKeys.data) {
+                helpers.addResult(results, 3,
+                    `Unable to list KMS keys: ${helpers.addError(listKeys)}`, region);
+                return rcb();
+            }
+
+            for (let forecastExportJob of listForecastExportJobs.data) {
+                if (!forecastExportJob && !forecastExportJob.Destination) {
+                    continue;
+                }
+
+                let { S3Config } = forecastExportJob.Destination;
+                let resource = forecastExportJob.ForecastExportJobArn;
+
+                if (S3Config && S3Config.KMSKeyArn) {
```

Comment:
Check if export is enabled and give passing result

Reply:
listForecastExportJobs will only list active export jobs that have been enabled by the User so we do not have to check it